test/e2e/run.sh

```diff
@@ -424,25 +458,71 @@ launch() { # script API
                 command-error "launching cri-resmgr failed, cannot find cri-resmgr PID"
             }
             ;;
+
+        "cri-resmgr-agent")
+            host-command "$SCP \"$adjustment_schema\" $VM_SSH_USER@$VM_IP:" ||
```

Comment:
Yeah... it seems that all `host-command "$SCP ...`'s in the project have either ugly-ish escaping or no quoting what so ever.

I'd probably leave this as is not to introduce the first hyphen in this context, and possibly have a clean-up commit that changes all double quotes with a single quote.

I'm not marking this conversation resolved in case you'd prefer some other way forward.

Reply:
> I'd probably leave this as is not to introduce the first hyphen in this context, and possibly have a clean-up commit that changes all double quotes with a single quote.

Yeah, sounds sensible